recipes/freealut/all/patches/freealut-1.1.0-conan-build.patch

```diff
@@ -0,0 +1,184 @@
+-
+-IF(UNIX)
+- SET(ADD_LIBS ${ADD_LIBS} m)
+-ENDIF(UNIX)
+-
+-SET(CMAKE_REQUIRED_INCLUDES ${OPENAL_INCLUDE_DIR})
+-CHECK_INCLUDE_FILES("AL/alc.h;AL/al.h" AL_HEADERS)
+-IF(NOT AL_HEADERS)
+-  MESSAGE(FATAL_ERROR "OpenAL header files not found - aborting.")
+-ENDIF(NOT AL_HEADERS)
+ 
+-IF(DEFINED OPENAL_INCLUDE_DIR)
+-  INCLUDE_DIRECTORIES(${OPENAL_INCLUDE_DIR})
+-ENDIF(DEFINED OPENAL_INCLUDE_DIR)
++FIND_PACKAGE(OpenAL REQUIRED)
++set(OPENAL_LIB "OpenAL::OpenAL")
+ 
+ FIND_PACKAGE(SleepFunction)
+ 
+@@ -131,30 +118,19 @@ SET(CMAKE_CXX_FLAGS "${ADD_CFLAGS} ${CMAKE_CXX_FLAGS}")
+ SET(CMAKE_SHARED_LINKER_FLAGS "${ADD_LDFLAGS} ${CMAKE_SHARED_LINKER_FLAGS}")
+ SET(CMAKE_MODULE_LINKER_FLAGS "${ADD_LDFLAGS} ${CMAKE_MODULE_LINKER_FLAGS}")
+ 
+-IF(BUILD_STATIC)
+-  # we can't create a static library with the same name
+-  # as the shared one, so we copy it over after creation
+-  ADD_LIBRARY(alut_static STATIC ${ALUT_SOURCES})
+-  TARGET_LINK_LIBRARIES(alut_static ${OPENAL_LIB} ${ADD_LIBS})
+-  IF(NOT WIN32)
+-    ADD_CUSTOM_COMMAND(
+-      TARGET alut_static
+-      POST_BUILD
+-      COMMAND ${CMAKE_COMMAND}
+-      ARGS -E copy
+-        ${CMAKE_BINARY_DIR}/${CMAKE_STATIC_LIBRARY_PREFIX}alut_static${CMAKE_STATIC_LIBRARY_SUFFIX}
+-        ${CMAKE_BINARY_DIR}/${CMAKE_STATIC_LIBRARY_PREFIX}alut${CMAKE_STATIC_LIBRARY_SUFFIX})
+-    INSTALL_FILES(/lib FILES ${CMAKE_STATIC_LIBRARY_PREFIX}alut${CMAKE_STATIC_LIBRARY_SUFFIX})
+-  ENDIF(NOT WIN32)
+-ENDIF(BUILD_STATIC)
+-
+-
+-ADD_LIBRARY(alut SHARED ${ALUT_SOURCES})
++ADD_LIBRARY(alut ${ALUT_SOURCES})
+ SET_TARGET_PROPERTIES(alut PROPERTIES VERSION ${VERSION} SOVERSION ${MAJOR_VERSION})
++set_target_properties(alut PROPERTIES PUBLIC_HEADER "include/AL/alut.h")
++if (NOT BUILD_SHARED_LIBS)
++  target_compile_definitions(alut PUBLIC ALUT_LIBTYPE_STATIC)
++endif()
+ TARGET_LINK_LIBRARIES(alut ${OPENAL_LIB} ${ADD_LIBS})
+ 
+-INSTALL_TARGETS(/lib alut)
+-INSTALL_FILES(/include/AL FILES include/AL/alut.h)
++install(TARGETS alut
++    LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
++    ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
++    RUNTIME DESTINATION ${CMAKE_INSTALL_BINDIR}
++    PUBLIC_HEADER DESTINATION ${CMAKE_INSTALL_INCLUDEDIR}/AL)
+ 
+ # needed for openal.pc.in and openal-config.in
+ SET(prefix ${CMAKE_INSTALL_PREFIX})
+@@ -170,8 +146,10 @@ CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/admin/pkgconfig/freealut.pc.in
+                ${CMAKE_BINARY_DIR}/admin/pkgconfig/freealut.pc @ONLY)
+ CONFIGURE_FILE(${CMAKE_SOURCE_DIR}/admin/pkgconfig/freealut-config.in
+                ${CMAKE_BINARY_DIR}/admin/pkgconfig/freealut-config @ONLY)
+-INSTALL_PROGRAMS(/bin FILES admin/pkgconfig/freealut-config)
+-INSTALL_FILES(/lib/pkgconfig FILES admin/pkgconfig/freealut.pc)
++install(PROGRAMS ${CMAKE_CURRENT_BINARY_DIR}/admin/pkgconfig/freealut-config 
++    DESTINATION ${CMAKE_INSTALL_BINDIR})
++install(FILES ${CMAKE_CURRENT_BINARY_DIR}/admin/pkgconfig/freealut.pc
++    DESTINATION ${CMAKE_INSTALL_LIBDIR}/pkgconfig)
```

Comment:
You can drop the pkgconfig installation as that needs to be removed later in the recipe anyway.

Reply:
done